modules/trusted-firmware-m/Kconfig

```diff
@@ -155,7 +199,7 @@ if BUILD_WITH_TFM
 config PM_PARTITION_SIZE_TFM_SRAM
 	hex
 	prompt "Memory reserved for TFM_RAM." if !TFM_PROFILE_TYPE_MINIMAL
-	default 0x10000 if TFM_PROFILE_TYPE_MINIMAL
+	default 0x8000 if TFM_PROFILE_TYPE_MINIMAL
 	default 0x16000 if SOC_NRF9160
 	default 0x40000 if SOC_NRF5340_CPUAPP
```

Comment:
260kB? What on earth is TF-M doing with all that RAM.

(Out-of-scope for this PR)